src/config/dark-sites.config

```diff
@@ -286,6 +297,8 @@ mynoise.net
 n-o-d-e.net
 namechk.com
 nanhu.ca
+neal.fun/size-of-space
```

Comment:
![](https://i.imgur.com/4K1aJSw.png)

Reply:
Page doesn't load when webgl is disabled.